### PR TITLE
Fix the alignment of report tiles in the finance report

### PR DESCRIPTION
**Pull request by Akshiteshnaithani**
Targets #2688 

### Description
I have Fixed the alignment of report tiles in the finance report.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
In you code changes, package.lock.json should not exist as it is not related to the issue. So remove that file from the code changes. 

In case you face issue with this, please take help from anyone near you.

File: Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php [hunk 1]
```diff
@@ -55,4 +56,6 @@
         </div>
     </div>
     </div>
+    </div>
+    </div>
```
Comment (GauravGusain98) [inline Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php:60]: The indentation looks incorrect here.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor suggestion. Please check

File: Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php (hunk 1 above)
Comment (Akshiteshnaithani) [reply Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php:60]: I dont get what do you mean by tiles ? for all box and their inside content padding are same .

Comment (GauravGusain98) [reply Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php:60]: These 2 gap looks different in size, No?
image

Comment (Akshiteshnaithani) [reply Modules/Report/Resources/views/finance/profit-and-loss/index.blade.php]: done

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**